app/client/components/holiday/holidaysOverview.tsx

```diff
@@ -116,7 +116,7 @@ const renderHolidayStopsOverview = (
       value={{
         ...holidayStopsResponse,
         reload: reloadWhichAlsoClearsAnyExistingHolidayStopToAmend,
-        existingHolidayStopToAmend
+        existingHolidayStopToAmend: existingHolidayStopToAmend || undefined
```

Comment:
the type required is `HolidayStopRequest | undefined` but `HolidayStopRequest | null` is supplied. Pre TypeScript upgrade this was allowed (as it has virtually the same effect) however TypeScript have tightened something up, so this just coerces `null` into `undefined`